default.nix

```diff
@@ -47,7 +47,7 @@ in {
              (compiler.callPackage ./iohk/default.nix {})
              (drv: {
                 executableToolDepends = [ pkgs.makeWrapper ];
-                libraryHaskellDepends = iohk-ops-extra-runtime-deps;
+                #libraryHaskellDepends = iohk-ops-extra-runtime-deps;
```

Comment:
This will break things, why is it commented out?

Reply:
Without this change when you (re)start `nix-shell` after changing cardano-sl revision it fails.

Here are redroduction steps:

1. Clone iohk-ops.
2. Launch nix-shell
3. Do `io set-rev cardanosl develop`
4. Exit the nix-shell and start it again.

After some time of building, you'll get a configuration error:
```
*** abort because of serious configure-time warning from Cabal
builder for ‘/nix/store/r1q3x2gzsncyh1a360f9zsryazycifkq-iohk-ops-0.1.0.0.drv’ failed with exit code 1
error: build of ‘/nix/store/r1q3x2gzsncyh1a360f9zsryazycifkq-iohk-ops-0.1.0.0.drv’ failed
```

Reply:
Can you paste the whole log? Error lies a while back.

Reply:
Here's the full log after running `nix-shell`
https://gist.github.com/hithroc/f02819dfd59b30fd21de7c7cbc692481

Reply:
Interesting, there are two different versions of scientific: https://gist.github.com/hithroc/f02819dfd59b30fd21de7c7cbc692481#file-log-L5788

Reply:
Submitted a fix in https://github.com/input-output-hk/cardano-sl/pull/2320

Reply:
Merged, if you bump develop this shouldn't be needed anymore.